web/public/locales/sl.json

```diff
@@ -49,8 +49,14 @@
     "renewable": "obnovljiva",
     "electricityproduction": "Proizvodnja električne energije",
     "electricityconsumption": "Poraba električne energije",
-    "bysource": "po viru",
-    "averagebysource": "povprečje glede na vir",
+    "by-source": {
+      "emissions": "Izpusti ogljika po virih",
```

Comment:
Changed from "po viru" to "po virih" as ChatGPT says this is more correct for this use case:

![Screenshot 2023-09-08 at 09 57 48](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/9992c1b3-86c0-4ead-b3cc-f8253f5d71cd)